recipes/blocked-matrix-utils/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "blocked-matrix-utils" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 550f5f7ee147b513abca70432cd85b0fe4fc87218e0447e9d538db33ec00d02d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - numpy
+    - python >=3
+    - pip
+  run:
+    - numpy
+    - python >=3
+
+test:
+  imports:
+    - util
+
+about:
+  home: "https://github.com/vahtras/util"
+  license: LGPL-2.1
```

Comment:
The license file in the repository doesn't match and is MIT, not GPL

Reply:
Thanks that was a mistake. Will change.